eng/common/scripts/Update-DocsMsPackages2.ps1

```diff
@@ -0,0 +1,129 @@
+<#
```

Comment:
Can we please avoid using "2" in the name? Lets give it a more meaningful name.

Reply:
Or is this only for testing and rollout with the goal of removing the 2?

Reply:
I see you mentioned this in your description but I would be interested in understanding more of your transition plan. Why not go ahead and remove the older version at the same time?

Reply:
I wasn't sure if we'd be able to get all 4 languages merged in one go so I wanted a way to keep things in both worlds if needed. "2" was superficially the easiest way to accomplish this. 

Since we have all 4 languages ready to merge we can get rid of the 2. The cleanup work item still stands as we'll need to delete old functions from `Language-Settings.ps1` which aren't called anymore.